### PR TITLE
Vsomeip selinux

### PR DESCRIPTION
**Pull request by Yarboa**
This pr tries to fix rpm build dependency in epel #244 
Submitted PR alone to verify selinux changes are not damaging tests

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -56,16 +56,17 @@ jobs:
...
+      - epel-9-x86_64
 
   - job: tests
     trigger: pull_request
     identifier: e2e-tiers
     tmt_plan: /plans/e2e/tier-0
     targets:
-    - centos-stream-9-x86_64
+      epel-9-x86_64:
+        distros: [centos-stream-9]
```
Comment (lsm5) [inline .packit.yaml:69]: I think you can (should) remove line 69. With line 69 I suspect it's using a pure-centos-stream and not centos-stream with epel enabled.

File: .packit.yaml [hunk 2]
```diff
@@ -78,10 +79,14 @@ jobs:
     identifier: e2e-ffi
     tmt_plan: /plans/e2e/ffi
     targets:
-    - centos-stream-9-x86_64
+      epel-9-x86_64:
+        distros: [centos-stream-9]
```
Comment (lsm5) [inline .packit.yaml:83]: same comment as line 69

File: .packit.yaml [hunk 3]
```diff
@@ -56,16 +56,16 @@ jobs:
...
+      - epel-9-ppc64le
+      - epel-9-x86_64
 
   - job: tests
     trigger: pull_request
     identifier: e2e-tiers
     tmt_plan: /plans/e2e/tier-0
     targets:
-    - centos-stream-9-x86_64
+    - epel-9-x86_64
```
Comment (lsm5) [inline .packit.yaml:68]: looks like the indentation is off here and also on line 81

**Comment by lsm5** [comment]
1. proc and sys mount issue, not sure. I'd suggest checking with the tmt folks. FWIW, I ran into a tmpfs size issue on container-selinux tests, and remounting worked fine for me.

2. no idea about the bluechictl failure. Is that persistent?

**Comment by Yarboa** [comment]
> 1. proc and sys mount issue, not sure. I'd suggest checking with the tmt folks. FWIW, I ran into a tmpfs size issue on container-selinux tests, and remounting worked fine for me.
> 2. no idea about the bluechictl failure. Is that persistent?

Persistent errors when using epel, 
on tmt it is working like charm.

**Comment by Yarboa** [comment]
Closing, #320 replace and try to suggest c9s gating

**Comment by Yarboa** [comment]
Closing, https://github.com/containers/qm/pull/320 replace and try to suggest c9s gating

**Comment by Yarboa** [comment]
Closing, https://github.com/containers/qm/pull/320 replace and try to suggest c9s gating

**Pull request closed without merging**